Dockerfile

```diff
@@ -5,7 +5,6 @@ ARG SESSION_CONTROL_CONF=session_control.conf
 WORKDIR /app
 
 COPY . .
-COPY agents/property.json.example agents/property.json
```

Comment:
before we support environment variable in property.json, i believe we still need property.json.example?

Reply:
no need property.json.example, running the server will not modify the contents of property.json.